sitemap.php

```diff
@@ -5,7 +5,7 @@
 // fixed, non-project pages to include in the listing with their
 // change frequency
 $fixed_pages = [
-    "default.php" => "monthly",
+    "index.php" => "monthly",
```

Comment:
Would it be easier to configure the apache to default in the container to default.php to avoid any risk to production?

Reply:
It's a good question. I think I'd rather finally get the codebase using the standard `index.php` everywhere instead of just most places with these two outliers. We can add redirects in the prod Apache config to redirect to `index.php` if someone actually uses them though to prevent possible issues on prod.